etc/kayobe/environments/aufn-ceph/kolla/config/bifrost/bifrost.yml

```diff
@@ -0,0 +1,12 @@
+---
+# Don't build an IPA deployment image, instead download upstream images.
+create_ipa_image: false
+download_ipa: true
+
+# Use a locally hosted CentOS8 cloud image.
+use_cirros: true
+{% if os_distribution == 'ubuntu' %}
+cirros_deploy_image_upstream_url: "https://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-amd64.img"
+{% else %}
+cirros_deploy_image_upstream_url: "https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-GenericCloud-8-20210210.0.x86_64.qcow2"
```

Comment:
We could probably go for the more modern https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-GenericCloud-8-20220913.0.x86_64.qcow2